lib/Dynamoose.ts

```diff
@@ -141,7 +142,7 @@ class Dynamoose {
       schema = new Schema(schema, options);
     }
 
-    const model = Model.compile(name, schema, options, this);
+    const model = compile(name, schema, options, this);
```

Comment:
Hmm, not a huge fan of this. Doesn't really say what it is compiling. Too broad in my opinion.

Reply:
With TS it will describe the output in fine detail and allow end users to know exactly what is happening. Do you want a more descriptive name?